cookbooks/clearwater/templates/default/shared_config.erb

```diff
@@ -10,6 +10,22 @@ sas_server=<%= @node[:clearwater][:sas_server] or "0.0.0.0" %>
 enum_server=<%= @node[:clearwater][:enum_server] or "" %>
 alias_list=<%= @alias_list %>
 
+<% if not @database.nil? %>
+# Storage Pool
+cassandra_hostname=<%= @database %>
+chronos_hostname=<%= @database %>
```

Comment:
@richardwhiuk , can you comment on this? 
I took the config as seen on your live nodes, and don't think these need ports, but i may be wrong